core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,50 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:600px){
+ .oppia-navbar-brand-name{
```

Comment:
Please follow the existing spacing and indentation conventions for this CSS file. (Look at the surrounding code, and mimic its style.)